docs/pages/configuration/pipelines/functions/exec_container.mdx

```diff
@@ -2,3 +2,41 @@
 title: Function exec_container
 sidebar_label: exec_container
 ---
+
+This function enables you to execute commands inside a container.
+
+## Arguments
```

Comment:
Whats missing here is the actual command that is provided by the arguments, so you would do for example:
```
exec_container --image-selector=ubuntu -- echo "Hello from within ubuntu container!"
```

Reply:
added to page